datastorage.lua

```diff
@@ -54,7 +54,7 @@ local function initDataDir()
     local sub_data_dirs = {
         "cache", "clipboard",
         "data", "data/dict", "data/tessdata",
-        "history", "ota",
+        "history", "ota", "plugins",
```

Comment:
If you're changing this line anyway, I don't think ota is relevant anymore on platforms where this function is relevant. Unless I'm mistaken @pazos?
```suggestion
        "history", "plugins",
```

Reply:
appImage?

Reply:
While I have considered looking into https://github.com/AppImage/AppImageUpdate (which iirc uses zsync?), I don't think that folder would be relevant to it. But right now it simply opens the link to the latest AppImage.